manifests/plugins/docker.pp

```diff
@@ -0,0 +1,57 @@
+class collectd::plugins::docker ( $modules ) {
+  validate_hash($modules)
+  Exec { path => [ '/bin/', '/sbin/', '/usr/bin/', '/usr/sbin/' ] }
+  include collectd
+
+  if $::osfamily == 'RedHat' {
+    exec { 'install epel-release':
+      command => 'yum install -y epel-release',
+      before  => Package['python-pip']
+    }
+  }
+
+  package { 'python-pip':
+    ensure => present,
+  }
+
+  exec { 'install dependencies':
+    command => 'pip install py-dateutil && pip install docker-py>=1.0.0 && pip install jsonpath_rw',
```

Comment:
that was a considered but not all plugins have the requirements file (or used that file differently).